backend/src/main/java/storage/TemporaryJVMStorage.java

```diff
@@ -1,6 +1,7 @@
 package storage;
 
 import model.Customer;
+import sun.reflect.generics.reflectiveObjects.NotImplementedException;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Use classes from the Java API instead of Sun classes. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1191)